core/base/ftmTree/FTMTreeUtils.cpp

```diff
@@ -1,4 +1,17 @@
 #include <FTMTree.h>
+#include <FTMTreeUtils.h>
+#include <iostream>
+
+void myPause() {
+  std::cout << "pause" << std::endl;
```

Comment:
you may want to remove that from the PR.
are you still using it?
(if you want to join this to the PR, I'd recommend to add that the `Debug` class, inherited by everybody)

Reply:
It is still used in the code but I will remove it in the next commit. It was more during testing (to stop when a problem was detected).